experiments/embedding_methods/bert_embedding.py

```diff
@@ -0,0 +1,33 @@
+import json
+from pathlib import Path
+
+import numpy as np
+from experiments.embedding_methods.embedding_interface import EmbeddingInterface
+from transformers import BertTokenizer, BertModel
+
+
+class BertTransformerEmbedding(EmbeddingInterface):
+
+    def __init__(self, model_name: str):
+        self.model = BertModel.from_pretrained(model_name)
+        self.model_name = model_name
+        self.tokenizer = BertTokenizer.from_pretrained(model_name)
+
+    def embed_sentence(self, sentence: str):
+        """
```

Comment:
this does not embed a sentence with bert. it tokenizes the input sentence(aka.) splits it into smaller chunks for example "koalabear"-> ["ko","ala", "bear"] then you generated an array that got you the ids belonging to those wordpieces. you didnt call the actual model

Reply:
https://mccormickml.com/2019/05/14/BERT-word-embeddings-tutorial/

Reply:
https://towardsdatascience.com/how-to-use-bert-from-the-hugging-face-transformer-library-d373a22b0209

Reply:
I updated the script to reflect the changes we talked about, but every time I try to run it my computer crashes. I think it's taking up too much memory on my computer, or I did something wrong?